Content.Server/Chemistry/Metabolism/OverdoseMetabolism.cs

```diff
@@ -0,0 +1,63 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Shared.Damage;
+using Content.Shared.Chemistry.Metabolizable;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Damage.Components;
+using Content.Server.Chemistry.Components;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Deals a certain amount of damage once a certain amount of reagent is ingested at once.
+    /// </summary>
+    [DataDefinition]
+    public class OverdoseMetabolism : IMetabolizable
+    {
+        /// <summary>
+        /// How much of the reagent should be metabolized each sec.
+        /// </summary> 
+        [DataField("rate")]
+        public ReagentUnit MetabolismRate { get; set; } = ReagentUnit.New(1);
+
+        /// <summary>
+        /// How much damage is changed when the overdose happenes, allowing for possible positive overdose?
+        /// </summary>
+        [DataField("healthChange")]
+        public float HealthChange { get; set; } = 500.0f;
```

Comment:
all of these datafields (except metabolismrate) should just be left empty by default and made nullable imo